### PR TITLE
fix: booker skeleton layout shift

### LINKED ISSUE #11614: [CAL-2550] booker: layout shift when changing months
https://github.com/calcom/cal.com/assets/8019099/eaecb69a-3d37-44a8-9729-9af08340bd10

<sub>[CAL-2550](https://linear.app/calcom/issue/CAL-2550/booker-layout-shift-when-changing-months)</sub>

### PR DESCRIPTION
**Pull request by sean-brydon**
Fixes layout shift in booker component

Fixes #11614

After: 

[CleanShot 2023-10-02 at 14.01.00.mp4](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/8j8kmxIa2LzQR4eN7JKh/b0d481b3-e114-429e-96b7-9bce720a4e0c/CleanShot%202023-10-02%20at%2014.01.00.mp4)

Before:

[CleanShot 2023-10-02 at 14.02.10.mp4](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/8j8kmxIa2LzQR4eN7JKh/4edd8e79-f790-4036-ac00-760ab869374e/CleanShot%202023-10-02%20at%2014.02.10.mp4)

### CODE REVIEW

**Comment by sean-brydon** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11653** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/11653" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/calcom/cal.com/11653?utm_source=stack-comment).

File: packages/features/bookings/Booker/components/AvailableTimeSlots.tsx [hunk 1]
```diff
@@ -103,22 +103,23 @@ export const AvailableTimeSlots = ({
...
-                availableMonth={
-                  dayjs(selectedDate).format("MM") !== dayjs(slots.date).format("MM")
-                    ? dayjs(slots.date).format("MMM")
-                    : undefined
-                }
-              />
-            ))}
+        {schedule.isLoading ? (
+          <div className="mb-3 h-8" />
+        ) : (
```
Comment (sean-brydon) [inline packages/features/bookings/Booker/components/AvailableTimeSlots.tsx:108]: We dont need to show a skeleton for the time header IMHO - leaving an empty block for it to load works just as well.

File: packages/features/bookings/components/AvailableTimes.tsx [hunk 2]
```diff
@@ -95,7 +95,7 @@ export const AvailableTimes = ({
 };
 
 export const AvailableTimesSkeleton = () => (
-  <div className="mt-8 flex w-[20%] flex-col only:w-full">
+  <div className="flex w-[20%] flex-col only:w-full">
```
Comment (sean-brydon) [inline packages/features/bookings/components/AvailableTimes.tsx:98]: Remove mt-8 as the skeleton was starting way too far down the page

**Review by PeerRich — APPROVED** [review]
great stuff

File: packages/features/bookings/components/AvailableTimes.tsx (hunk 2 above)
Comment (PeerRich) [reply packages/features/bookings/components/AvailableTimes.tsx:98]: only question is, if we should increase the height for column view skeletons

I think the height is slightly different right now

**Pull request merged**